torch/nn/modules/module.py

```diff
@@ -2649,13 +2654,16 @@ def named_parameters(
         )
         yield from gen
 
-    def buffers(self, recurse: bool = True) -> Iterator[Tensor]:
+    def buffers(
+        self, recurse: bool = True, *, persistent: bool = False
+    ) -> Iterator[Tensor]:
         r"""Return an iterator over module buffers.
 
         Args:
-            recurse (bool): if True, then yields buffers of this module
+            recurse (bool, optional): if True, then yields buffers of this module
                 and all submodules. Otherwise, yields only buffers that
                 are direct members of this module.
+            persistent (bool, optional): if True, then yields persistent buffers only. Defaults to False.
```

Comment:
can we make this an enum with 3 values that returns either `PERSISTENT`, `NON_PERSISTENT` or `ALL`?

Reply:
@mikaylagawarecki I pushed the 3-valued options as a separate PR: #132994